test/e2e/vault-decryption-chrome.spec.js

```diff
@@ -0,0 +1,115 @@
+const { strict: assert } = require('assert');
+const path = require('path');
+const fs = require('fs');
+const {
+  withFixtures,
+  WALLET_PASSWORD,
+  openSRPRevealQuiz,
+  completeSRPRevealQuiz,
+  tapAndHoldToRevealSRP,
+  completeCreateNewWalletOnboardingFlowWithOptout,
+} = require('./helpers');
+
+const VAULT_DECRYPTOR_PAGE = 'https://metamask.github.io/vault-decryptor';
```

Comment:
In general we try to keep real network requests out of PR status checks so that outages or website changes don't break our CI.

Ideally we'd use an offline version of the vault decryptor. But we have no offline version at the moment. We could get one by publishing the page to npm, similar to how we handle the test dapp.

Alternatively (or additionally?) we could test the live page but only run this on `develop` and/or releases candidates, so that it doesn't interrupt development when it breaks due to page changes

Thoughts?

Reply:
Yeah, making the test runnable offline makes sense, as long as we can run locally a version of the vault decryptor that is guaranteed to be the same as the published one.

It would be nice to have it built and published by the CI along with updating the github-pages' deployed version - I see that the test dapp also gets published manually, but in the vault decryptor case the process is quite odd because it assumes that the `bundle.js` is manually built and updated in the same PR of the changes.
 
Perhaps we can add the CI publish step, along with the build

Reply:
> Alternatively (or additionally?) we could test the live page but only run this on develop and/or releases candidates, so that it doesn't interrupt development when it breaks due to page changes

This makes sense, I would:
- Leave the test run on PRs, but with an offline page
- Add a real test against the live page, on `develop` and RCs

This way we would be covered against cases of GitHub service disruption _and_ cases of misalignment between the Metamask vault and what the vault decryptor page expects

Reply:
Makes sense! Perhaps we can just do the second one for now, in this PR, so that we can merge this while working on publishing the vault decryptor

Reply:
Applied in ac88ed9c94f6b45c3596f9c0b84dd696f1de7f54! (not sure how we can test this out though)

Reply:
cc @danjm @chloeYue does this strategy make sense to you?

Reply:
cc @danjm @chloeYue @brad-decker @pedronfigueiredo does this strategy make sense to you?